infra/base-images/base-runner/reproduce

```diff
@@ -15,8 +15,9 @@
 #
 ################################################################################
 
-FUZZER=$1
-shift
+DEBUGGER=$1
```

Comment:
Can we change this to just be an env var rather than taking in a full argument (through -e argument for docker)? Passing '' feels very hacky when we don't want a debugger.